applications/sentence-transformers-ft/eval.py

```diff
@@ -0,0 +1,99 @@
+from datasets import load_dataset
+import modal
+from modal import Image
+from sentence_transformers import InputExample, evaluation, SentenceTransformer
+import csv
+
+DATASET_ID = "quora"
+GPU_CONFIG = "a10g"
+MODEL_IDS = [
+    "BAAI/bge-small-en-v1.5",
+    "BAAI/bge-base-en-v1.5",
+    "thenlper/gte-small",
+    "intfloat/e5-small-v2",
+    "sentence-transformers/all-MiniLM-L12-v2",
+    "sentence-transformers/all-mpnet-base-v2",
+    "sentence-transformers/all-distilroberta-v1",
+    "sentence-transformers/multi-qa-mpnet-base-dot-v1",
+    "sentence-transformers/multi-qa-distilbert-cos-v1",
+    "llmrails/ember-v1",
```

Comment:
any jina models?

Reply:
running that rn

Reply:
Getting a weird error with specifically Jina, some remote code error I think which makes it broken on Modal

```
File "/usr/local/lib/python3.11/site-packages/transformers/dynamic_module_utils.py", line 611, in resolve_trust_remote_code
    raise ValueError(
ValueError: The repository for /root/.cache/torch/sentence_transformers/jinaai_jina-embeddings-v2-base-en/ contains custom code which must be executed to correctly load the model. You can inspect the repository content at https://hf.co//root/.cache/torch/sentence_transformers/jinaai_jina-embeddings-v2-base-en/.
Please pass the argument `trust_remote_code=True` to allow custom code to be run.

Stopping app - uncaught exception raised locally: ValueError('The repository for /root/.cache/torch/sentence_transformers/jinaai_jina-embeddings-v2-base-en/ contains custom code which must be executed to correctly load the model. You can inspect the repository content at https://hf.co//root/.cache/torch/sentence_transformers/jinaai_ji
```